lib/plugins/rest.js

```diff
@@ -43,7 +45,34 @@ exports.register = function (server, options, next) {
 				return awsAdapter.runInstances(instance);
 			})
 			.then(function (instance) {
+				var updatedInstance;
+
 				reply(instance).created("/v1/instances/" + instance.id);
+				awsInstanceID = instance.instanceID;
```

Comment:
It looks like this variable isn't used outside of this block. Is there a reason that it needs to be scoped at the handler level?